Documentation/Signoffs/Motor System.md

```diff
@@ -83,59 +83,45 @@ The standard NEC 705.12(D)(2) Bus or Conductor Ampere Rating states "One hundred
-
 ## Schematic
 
-![Schematic](https://github.com/DillonSW/Capstone_Team_5/blob/Team5-signoff-Motor-System/images/DriverAndMotorRevisionFive.jpg)
-
-The schematic above shows the wiring connections between the microcontroller/microcomputer, driver, and motor. The encoder is a part of the driver and has its own output ports on the driver, however it is labeled as separate in the datasheets. The resistor options for connecting the microcontroller/microcomputer with the driver are: 0 for 5V, 1kΩ for 12V, or 2kΩ for 24V. Because there are 5 resistor connections, 10, 2kΩ resistors would suffice for covering all of them, in which we can use 2, 2kΩ resistors in parallel to create a 1kΩ connection.
-
-Also to note, the HSS86 driver can take between 20-80 AC, or 30-110 DC voltage. Because the wall outlet's nominal voltage is between 110-120 AC voltage, we must step it down in order to safely power the driver.
-^ Source: https://www.jbcnc.se/images/datasheets/HSS86.pdf
-^ Source: https://conquerallelectrical.ca/us-electrical-outlet-voltage/?utm_source=rss&utm_medium=rss&utm_campaign=us-electrical-outlet-voltage
-
-To adhere to the NEC 310-16 table, the following must be considered:
-
-* The step-down transformer, based on constraints, **must** output 24VAC to the driver.
+![Schematic](https://github.com/DillonSW/Capstone_Team_5/blob/Team5-signoff-Motor-System/images/DriverAndMotorRevisionSeven.jpg)
 
-* The driver's **peak current draw** is 8A.
+The schematic above shows the wiring connections between the microcontroller/microcomputer, driver, and motor. The resistor options for connecting the microcontroller/microcomputer with the driver are: 0 for 5V, 1kΩ for 12V, or 2kΩ for 24V. Because there are 5 resistor connections, 10, 2kΩ resistors would suffice for covering all of them, in which we can use 2, 2kΩ resistors in parallel to create a 1kΩ connection.
 
-This means the secondary side of the transformer will draw, at max, 192W.
+This selected dirver and motor will be able to connect and work together. The driver, STP-DRVAC-24025, states "Only use stepper motors wound for high bus voltages (**STP-MTRAC** motors) with high bus voltage drives (STP-DRVAC-xxxxx)." The motor we chose follows this naming convention: **STP-MTRAC**-34156.
 
-Using the equation:
+Also to note, the STP-DRVAC-20425 driver can take a nominal input voltage of 90-240VAC. Because of this we will not need a step-down transformer.
+^ Source: https://www.automationdirect.com/adc/shopping/catalog/motion_control/stepper_systems/stepper_drives/stp-drvac-24025
+^ Source: https://www.automationdirect.com/adc/shopping/catalog/motion_control/stepper_systems/single_shaft_stepper_motors/stp-mtrac-34156
 
-$Pin = Pout$
+The fuses labeled in the schematic are AGC4 fuses. These are "Bussmann fuse, AGC series, small dimension, fast-acting, 4A, 250 VAC, glass tube." This protects the system for 160% of the rated current **and** are the manufacturer recommended fuses.
```

Comment:
I recommend using breakers instead (the same rating as the fuses are fine). Fuses require fuse holders which aren't in the design yet (so there's no additional work to swap to breakers).